KsApi/models/Backing.swift

```diff
@@ -68,12 +91,12 @@ extension Backing: Argo.Decodable {
  on that environment. This is a workaround to allow us to test on Staging and should be deleted once the
  data is being returned normally.
  */
-private func tryDecodePaymentSource(_ json: JSON?) -> Decoded<GraphUserCreditCard.CreditCard?> {
+private func tryDecodePaymentSource(_ json: JSON?) -> Decoded<Backing.PaymentSource?> {
```

Comment:
Do we still need this in this PR?

Reply:
We do. I tried removing it, but same issue still happens on Staging.